docs/quality-assurance/manual-tests/workflow-accept-relation.md

```diff
@@ -0,0 +1,235 @@
+
+<img src="https://user-images.githubusercontent.com/53532851/93903328-26c87280-fd16-11ea-90a9-e05682c99105.jpeg" alt ="registration" width="200"/>
+
+##### EXPECTED RESULT: 
+"Please choose atleast one of the following options"
+##### ACTUAL RESULT: 
+"Please choose atleast one of the following options"
+
+
+
+## **Test-2 : Email confirmation**
+confirming email after successfully signing up
+
+
+#### *test 2.1: email confirmed*
+<img src="https://user-images.githubusercontent.com/53532851/93903499-5a0b0180-fd16-11ea-9bcb-d22752a3dfd9.png" alt ="registration" width="800"/>
+<img src="https://user-images.githubusercontent.com/53532851/93903525-62fbd300-fd16-11ea-9d76-29c7d389b076.png" alt ="registration" width="800"/>
+
+##### EXPECTED RESULT: 
+"you have confirmed your account"
+##### ACTUAL RESULT: 
+"you have confirmed your account"
+
+
+
+## **Test-3 : Login**
+logging in the account
+
+
+> ### **positive tests**
+
+#### *test 3.1: login via verified email address :*
+
+<img src="https://user-images.githubusercontent.com/53532851/93901760-7b6aee00-fd14-11ea-8e07-8728d96af082.gif" alt ="registration" width="200"/>
+
+##### EXPECTED RESULT: 
+"Logging successful"
+##### ACTUAL RESULT: 
+"Logging successful"
+
+#### *test 3.2: login via username :*
+
+<img src="https://user-images.githubusercontent.com/53532851/93901717-7148ef80-fd14-11ea-96ef-7d2911619eba.gif" alt ="registration" width="200"/>
+
+##### EXPECTED RESULT: 
+"Logging successful"
+##### ACTUAL RESULT: 
+"Logging successful"
+
+
+> ### **negative tests**
+
+#### *test 3.3: entering wrong password :*
+
+<img src="https://user-images.githubusercontent.com/53532851/93903799-c38b1000-fd16-11ea-8db1-bf8f36a64670.jpeg" alt ="registration" width="200"/>
+
+##### EXPECTED RESULT: 
+"password incorrect"
+##### ACTUAL RESULT: 
+`"something went wrong"`
+
+
+#### *test 3.4: logging before confirming email :*
+
+<img src="https://user-images.githubusercontent.com/53532851/93914882-ca208400-fd24-11ea-923c-7af3a700d1c1.gif" alt ="registration" width="200"/>
+
+##### EXPECTED RESULT: 
+"please verify your email"
+##### ACTUAL RESULT: 
+"please verify your email"
+
+
+
+## **Test-4 : sending mentorship request**
+sending request to be a mentor or mentee to another user
+
+
+> ### **positive**
+
+#### *test 4.1: sending request to a user :*
+
+<img src="https://user-images.githubusercontent.com/53532851/93903804-c4bc3d00-fd16-11ea-8752-c0d296030de4.jpeg" alt ="registration" width="200"/>
+
+##### EXPECTED RESULT: 
+"mentorship relation was sent successfully"
+
+
+##### ACTUAL RESULT: 
+"mentorship relation was sent successfully"
+
+> ### **negative**
+
+#### *test 4.2: sending request to the same user again :*
+
+<img src="https://user-images.githubusercontent.com/53532851/93901623-5a0a0200-fd14-11ea-9beb-4cdfba5a1eb0.gif" alt ="registration" width="200"/>
+
+##### EXPECTED RESULT: 
+"mentorship relation request already sent"
+
+
+##### ACTUAL RESULT: 
+`"mentorship relation was sent successfully"`
```

Comment:
This is what you were trying to explain in the comments, now I get it ! Yes, please open an issue for this. 

Did you try the test where 5 users sends requests to the same mentor user? It maybe a good idea to open a feature issue for this, so that the maintainers could discuss and decide.

Reply:
I only made three accounts so i was no not able to perform test for many users ,sending requests to the same user but  I think all of them will be able to send requests , sure I will create a feature issue for this
thankyou : )